common/models/model-definition.json

```diff
@@ -96,7 +96,8 @@
     "methods": {
       "embed": {
         "as": "object",
-        "key": "name"
+        "key": "name",
```

Comment:
`key` has to be set as well otherwise the property doesn't set the name property to the method name.